process/miniblocks/miniblocksProcessor.go

```diff
@@ -18,16 +18,18 @@ import (
 var log = logger.GetOrCreate("indexer/process/miniblocks")
 
 type miniblocksProcessor struct {
-	hasher      hashing.Hasher
-	marshalier  marshal.Marshalizer
-	selfShardID uint32
+	importDBMode bool
```

Comment:
re-organize the members. place the uin32 and the bool one near each other (last 2 positions) for memory alignment

Reply:
Will do in the next PR